Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -103,19 +105,15 @@ public struct TargetSourcesBuilder {
 
     @discardableResult
     private func validTargetPath(at: AbsolutePath) -> Error? {
-        guard let cwd = self.fs.currentWorkingDirectory else {
-            return StringError("Unknown Current Working Directory")
-        }
-        
         // Check if paths that are enumerated in targets: [] exist
         guard self.fs.exists(at) else {
-            return StringError("\(at) could not be found")
+            return StringError("File not found")
         }
 
         // Excludes, Sources, and Resources should be found at the root of the package and or
         // its subdirectories
-        guard at.pathString.hasPrefix(cwd.pathString) else {
-            return StringError("\(at) should be found within the root of \(cwd)")
+        guard at.pathString.hasPrefix(self.packagePath.pathString) else {
+            return StringError("'\(self.packagePath.pathString)' should contain: ")
```

Comment:
Should this say some thing like "File must be within the package directory structure" (@abertelrud may have nicer way to phrase it)